physionet-django/console/templates/console/console_home.html

```diff
@@ -0,0 +1,12 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}{{ SITE_NAME }} - Admin Console{% endblock %}
+
+{% block content %}
+<h1>Admin Console</h1>
+<hr>
+{% lorem 3 p %}
```

Comment:
@alistairewj
Do we want to keep the new admin console home page? If yes, could you please provide a text for that?

Reply:
I think a home page makes sense. Not sure what text to put, maybe something generic like:

"Welcome to the PhysioNet console. This area allows privileged users to update and manage content on PhysioNet." (of course using the debranded variable there)

Reply:
Thanks!